src/main/java/nextstep/courses/domain/CoverImage.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.courses.domain;
+
+import java.util.Objects;
+
+public class CoverImage {
+
+    private static final int IMAGE_SIZE_LIMIT_BYTE = 1024000000;
```

Comment:
자바에서는 콤마(,)대신 언더바를 통해 숫자의 가독성을 높혀줄 수 있습니다!

```suggestion
    private static final int IMAGE_SIZE_LIMIT_BYTE = 1_024_000_000;
```